packages/eslint-plugin-circuit-ui/vitest.config.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2023, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    globals: true,
```

Comment:
Why are we relying on globals in this package and not the others?

Reply:
It's enabled in the ESLint plugin and in Circuit UI. From the docs:

> Jest has their [globals API](https://jestjs.io/docs/api) enabled by default. Vitest does not. You can either enable globals via [the globals configuration setting](https://vitest.dev/config/#globals) or update your code to use imports from the vitest module instead.
> 
> If you decide to keep globals disabled, be aware that common libraries like [testing-library](https://testing-library.com/) will not run auto DOM [cleanup](https://testing-library.com/docs/svelte-testing-library/api/#cleanup).

The ESLint plugin test utils time out without the globals being defined. I enabled the globals in Circuit UI to ensure `testing-library` works properly and also added the explicit imports because I've always felt that the globals are too much magic.